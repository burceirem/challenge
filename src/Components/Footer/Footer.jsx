import React from 'react';
import './Footer.css';

const Footer = () => {

    return (
        <div data-testid="footer" className="footer"> 
          <p>PANKOD CHALLENGE PROJECT </p>
        </div>

    );
};

export default Footer;