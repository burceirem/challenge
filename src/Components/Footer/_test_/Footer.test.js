import React from 'react';
import { render  } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

it ( "renders footer component correctly", () => {
    const { getByTestId } = render(<Footer />);
    expect(getByTestId("footer")).toHaveClass("footer"); 
});



it("matches snapshot",() => {
 const tree = renderer.create(<Footer/>);
 expect(tree).toMatchSnapshot();
});

