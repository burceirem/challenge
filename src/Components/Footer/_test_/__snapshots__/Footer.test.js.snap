// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="footer"
  data-testid="footer"
>
  <p>
    PANKOD CHALLENGE PROJECT 
  </p>
</div>
`;
